src/main/java/com/app/katchup/Meeting/MeetingService.java

```diff
@@ -18,17 +19,26 @@
     @Autowired
     MeetingRepository meetingRepository;
 
-    public Meeting createMeeting(Meeting meeting){
+    public Meeting createMeeting(Meeting meeting) throws Exception{
         if(meetingRepository.findMeetingByFilter(meeting.getHost(), meeting.getStartDateTime(),
                                                     meeting.getEndDateTime(), meeting.getVenue()) == null){
-           meeting = meetingRepository.save(meeting);
-           return meeting;
+            if(isDateTimeValid(meeting)){
+                meeting = meetingRepository.save(meeting);
+                return meeting;
+            }else {
+
+                    throw new Exception("startDateTime is before EndDateTime or date is not valid");
+            }
         }
         else{
             throw new EntityExistsException("Sorry! Meeting cannot be created. There exists another meeting on the given data");
         }
     }
 
+    public MeetingService() {
+        super();
```

Comment:
what is this for ? 😕

Reply:
Actually I didnt put that, Anyway will remove